sdk/resources/mgmt/src/test/java/com/azure/management/resources/core/TestBase.java

```diff
@@ -164,6 +164,23 @@ public void beforeTest(TestInfo testInfo) throws IOException {
         final String skipMessage = shouldCancelTest(isPlaybackMode());
         Assumptions.assumeTrue(skipMessage == null, skipMessage);
 
+        String skipOutput = System.getProperty("skipOutput");
+        if (isPlaybackMode() && skipOutput != null && skipOutput.equalsIgnoreCase("true")) {
+            System.setOut(new PrintStream(new OutputStream() {
+                @Override
+                public void write(int b) throws IOException {
+
```

Comment:
empty?

Reply:
Yes, just skip the output. So much output will make playback error in devops.